index.html

```diff
@@ -207,15 +207,15 @@ <h2 class="about-title">Skills</h2>
       <h2 class="contact-h2">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
       <form id="form"  action="https://formspree.io/f/mbjelbzj" method="POST">
         <div class="input-control">
-            <input id="fullname" name="fullname" type="text" placeholder="Full name" maxlength="30">
+            <input id="fullname" name="fullname" type="text" placeholder="Full name" maxlength="30" aria-label="fullname">
             <div class="error"></div>
         </div>
         <div class="input-control">
-            <input id="email" name="email" type="text" placeholder="Email address">
+            <input id="email" name="email" type="text" placeholder="Email address" aria-label="emailaddress">
             <div class="error"></div>
         </div>
         <div class="input-control">
-          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500"></textarea>
+          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500" aria-label="message"></textarea>
           <div class="error"></div>
       </div>
         <button class= "box-button about-btn" type="submit">Get in touch</button>
```

Comment:
- [ ] Please kindly adjust the contrast of **all your button elements**, I observed that the color is too close to the background-color. Kindly work on this in order to enhance the user accessibility of your app. 👍 You can use this [tool](https://www.aditus.io/button-contrast-checker/) as suggested earlier.